pkg/profile/update.go

```diff
@@ -43,7 +57,7 @@ func Update(
 	if bundlesPtr == nil {
 		bundles = CommaSeparatedFromStringList(prof.Spec.Bundles)
 	} else {
-		bundles = *bundlesPtr
+		bundles = CommaSeparatedFromStringList(bundlesPtr)
```

Comment:
I find the name `CommaSeparatedFromStringList` hard to parse. Does `GetConcatenatedStringFromBundlesList` or `ConcatenateBundlesFromList` make more sense?